src/tests/JIT/opt/Multiply/IntMultiply.csproj

```diff
@@ -11,7 +11,7 @@
       <HasDisasmCheck>true</HasDisasmCheck>
     </Compile>
 
-    <CLRTestEnvironmentVariable Include="COMPlus_TieredCompilation" Value="0" />
-    <CLRTestEnvironmentVariable Include="COMPlus_JITMinOpts" Value="0" />
+    <CLRTestEnvironmentVariable Include="DOTNET_TieredCompilation" Value="0" />
+    <CLRTestEnvironmentVariable Include="DOTNET_JITMinOpts" Value="0" />
```

Comment:
Yes, since it will force JTMinOpts to always be `0` even when the test CI leg sets JITMinOpts to `1`. We haven't done enough work to try to exclude certain tests based on the environment variables they rely on.

Reply:
It's still a little odd to me, `COMPlus_` is an accepted prefix to the environment variables, so why doesn't the old alias work to force min-opts?

Reply:
Because there are two of the same env variables at play? What happens if "DOTNET_JitMinOpts=1" and "COMPlus_JItMinOpts=0" are set at the same time?

Reply:
Ah I see, so the problem is happening due to both of them being set to different values. That makes sense.